docker-compose.yml

```diff
@@ -0,0 +1,80 @@
+version: '3.7'
+networks:
+  dspacenet:
+services:
+  dspace:
+    container_name: dspace
+    image: "${DOCKER_OWNER:-dspace}/dspace:${DSPACE_VER:-dspace-7_x-jdk8-test}"
+    build:
+      context: .
+      dockerfile: Dockerfile.jdk8-test
+    depends_on:
+    - dspacedb
+    networks:
+      dspacenet:
+    ports:
+    - published: 8080
+      target: 8080
+    stdin_open: true
+    tty: true
+    volumes:
+    - assetstore:/dspace/assetstore
+    - ./dspace/src/main/docker-compose/local.cfg:/dspace/config/local.cfg
+  dspace-angular:
```

Comment:
It's slightly confusing that the Docker configurations in `DSpace/DSpace` should kick off the Angular UI (as the Angular UI is not in this project, it's in the separate `DSpace/dspace-angular` project).  I'd almost expect that `DSpace/DSpace` should only include Docker configurations to kick off the REST API, while `DSpace/dspace-angular` should include the Docker configurations to kick off the Angular UI. But, I imagine that'd complicate things here.